python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_environment.py

```diff
@@ -0,0 +1,31 @@
+def test_environment_with_container_context(
```

Comment:
Can you also add a test for setting the environment via instance config? Similar to https://github.com/dagster-io/dagster/blob/f305b65e057eb73c6be70ebb430c608941a62e40/python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_secrets.py#L9